R/tm_g_distribution.R

```diff
@@ -121,7 +131,8 @@ tm_g_distribution <- function(label = "Distribution Module",
                               plot_height = c(600, 200, 2000),
                               plot_width = NULL,
                               pre_output = NULL,
-                              post_output = NULL) {
+                              post_output = NULL,
+                              decorators = list(default = teal_transform_module())) {
```

Comment:
*Topic to discuss*

Why are we defaulting to an empty transform module? The list should be empty!